lang/src/gen/abi.rs

```diff
@@ -137,6 +138,13 @@ fn generate_abi_messages<'a>(
 ) -> impl Iterator<Item = TokenStream2> + 'a {
     contract.messages.iter().map(|message| {
         let selector = message.selector();
+        let selector = format!(
+            "[{}, {}, {}, {}]",
+            selector[0], selector[1], selector[2], selector[3]
+        );
+        let selector = parse_str::<syn::Expr>(&selector)
+            .expect("failed to parse selector");
+
```

Comment:
These transformations are required because `syn` doesn't provide a `ToTokens` impl for `[u8; 4]`, right?

Reply:
Yup, right! I searched for a more elegant solution, but haven't found one yet.